python/lsst/ts/m2/csc.py

```diff
@@ -299,6 +316,14 @@ async def _process_telemetry(self, message: dict | None = None) -> None:
 
         # Publish the SAL telemetry
         if isinstance(message, dict):
+            # Bypass the following telemetry before we support them in ts_xml
+            if message["id"] in (
```

Comment:
Correct me if I am wrong but `_publish_message_by_sal` already checks if the message translates into a topic and handles publishing/warning in each condition. Why not just rely on this feature to give you automatic backward/forward compatibility? This way you won't need to change the code later at the expense of having a few additional warning messages published in the meantime.

Reply:
The `_publish_message_by_sal ()` will `log` a lot of warnings for telemetry and I try to avoid this in the current debugging stage:

```
            self.log.warning(
                f"Unspecified message: {message_name_original}, ignoring..."
            )
```

As I mentioned in the comment, I will remove this part of code after `ts_xml` supports these new events and telemetry.

Thanks!